README.md

```diff
@@ -48,20 +48,40 @@ or
 ```sh
 $ yarn add -D sr-commit-analyzer sr-release-notes-generator conventional-changelog-eslint
 ```
+Then, create  a `release.config.js` file in a `config` folder in the root folder of your project :
+```js
+/* eslint-disable no-useless-escape */
+module.exports = {
+  analyzeCommits: {
+    preset: 'eslint',
+    releaseRules: './config/release-rules.js', // optional, only if you want to set up new/modified release rules inside another file
+    parserOpts: { // optional, only you want to have emoji commit support
+      headerPattern: /^(?:\:(\w*)\:)?\s(\w*)\:\s(.*?)(?:\((.*)\))?$/,
```

Comment:
I do not understand the overhead of this regex (also emojis with `-` in it wouldn't match). I thing a better one with less overhead would be following:

```js
/^(?::([\w-]*):)?\s*(\w*):\s*(.*)$/
```

If I missed anything in your regex, just let me know. What actually does `semantic-release` when the commit is multiline? E.g.:

```
:hammer: Refactor: My commit

My body with more information
```

Will this information be dropped by `semantic-release`? If not, the information after a newline should be added as well and the regex should look like this:

```js
/^(?::([\w-]*):)?\s*(\w*):\s*([\s\S]*)$/ // just the last matching group is different
```

Reply:
If you take a look to the original [regex](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-eslint/parser-opts.js#L4)  of eslint: 

```js
headerPattern: /^(\w*)\: (.*?)(?:\((.*)\))?$/
```
I just updated this one : I am not a regex fan ^^
Semantic-release just rely on the package that does the work XD : nothing more than that. This question should be asked to conventional-changelog guys

Reply:
Ah alright! Then please change the regex to my first suggested one. I will tweak their regex as well 👍 

> The complete list of github emoji has also some _ in the name

Yap that is correct, but `\w` is the same as `[a-zA-Z_]`, which also includes underscores